deploy-service/common/pom.xml

```diff
@@ -207,6 +207,12 @@
             <groupId>org.quartz-scheduler</groupId>
             <artifactId>quartz</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.28</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Great! Can you also extract the version to the parent POM and update the universal lib as well?

Reply:
Good idea! I'm really out of practice with Maven...